### PR TITLE
Hide NavBar on Cleezy page

### PR DESCRIPTION
**Pull request by pnavab**

### CODE REVIEW

File: src/Pages/URLShortener/URLShortener.js [hunk 1]
```diff
@@ -107,7 +107,10 @@ export default function URLShortenerPage(props) {
 
   return (
     <div className='url-shortener'>
-      <Header title="Welcome to the url Shortener!!" />
+      <br></br>
+      <br></br>
+      <br></br>
+      <br></br>
```
Comment (evanugarte) [inline src/Pages/URLShortener/URLShortener.js:113]: do you still need this?

**Review by evanugarte — COMMENTED** [review]
update this branch when you can

https://github.com/SCE-Development/Clark/wiki/FAQ#how-do-i-update-my-branch

File: src/Routing.js [hunk 2]
```diff
@@ -63,14 +63,14 @@ export default function Routing({ appProps }) {
...
-    //   inAdminNavbar: true
-    // },
+    //
+    {
+      Component: EmailPage,
+      path: '/email-list',
+      allowedIf: userIsOfficerOrAdmin,
+      redirect: '/',
+      inAdminNavbar: true
+    },
```
Comment (evanugarte) [inline src/Routing.js:73]: undo this change, this should still be commented out

**Review by evanugarte — APPROVED** [review]

File: src/Pages/URLShortener/URLShortener.js [hunk 3]
```diff
@@ -66,19 +66,19 @@ export default function URLShortenerPage(props) {
     }
   }
 
+  async function maybeSubmitUrlOnKeyPresses(e) {
+    if ((e.ctrlKey || e.metaKey) && (e.key === 'Enter')) {
+      maybeSubmitUrl(url);
```
Comment (danknessdra) [inline src/Pages/URLShortener/URLShortener.js:71]: Maybe make the resetting of fields in this function, not in the function this references (Code orthogonality)

**Review by danknessdra — APPROVED** [review]
